Gordon360/Models/ViewModels/RecIM/SportViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Gordon360.Models.CCT;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class SportViewModel
+    {
+        public int? ID { get; set; }
+        public string? Name { get; set; }
+        public string? Description { get; set; }
+        public string? Rules { get; set; }
+        public string? Logo { get; set; }
```

Comment:
I wasn't completely sure how to handle this. SportViewModel is identical to the SQL model as well as the fact that being incredibly similar to its upload and patch viewmodel. So I thought it might be an okay implementation to just have a single viewmodel for sport. However, I could split them up into upload and patch if it makes it clearer.

Reply:
Yeah, splitting them up is definitely preferred.

Reply:
Wait, I just realized they're already split. There's a chance we had this conversation before, then I forgot to un-null them.